frontend/src/components/ShootDns/GDnsProvider.vue

```diff
@@ -28,17 +28,19 @@ SPDX-License-Identifier: Apache-2.0
         />
       </v-chip>
     </template>
-    <v-list class="pa-0">
+    <v-list>
```

Comment:
we should increase the min-width of the popover as the title could get cut off
![Screenshot 2023-09-14 at 15 20 01](https://github.com/gardener/dashboard/assets/5526658/6de2f650-3cd2-4730-a418-31323accf941)

Reply:
done